### ISSUE TITLE
Feature Request: Add strict-ssl configuration item.

### ISSUE THREAD

**Issue by amumu2000** [oven-sh/bun#9849]
### What is the problem this feature would solve?

I used the internal NPM source of the company, and then I configured it in the bunfig.toml of the project.
```toml
# ./bunfig.toml
[install]
registry = "xxxxxx"
```
But, I encountered the same problem as using the NPM package manager.
**UNABLE_TO_VERIFY_LEAF_SIGNATURE**
If using NPM, I can edit the. npmrc file to solve the problem.The solution is to set `strict-ssl` to false.
```
# ~./.npmrc
strict-ssl=false
```
I carefully searched the documentation and found that bun does not have this configuration.

### What is the feature you are proposing to solve the problem?

Add strict-ssl configuration item.

### What alternatives have you considered?

none

**Comment by GavanHub** [comment]
I have the same problem

**Comment by xblurx** [comment]
yeah its crucial when the package management system in your organization runs through a proxy that additionally stores some internal packages

**Comment by DSK-QwQ** [comment]
Plus one on this one!
It's a litteral pain the *** that I cannot disable strict_ssl on internal basis, and this actually disables the use of bun in my project.

**Comment by yeka** [comment]
Is there a workaround to this issue?

**Comment by micodev** [comment]
same here

**Comment by ardyfeb** [comment]
+1
